commit 43383e0adbd79b2f6847298640f619565e37ad72
Author: Panu Matilainen <pmatilai@redhat.com>
Date:   Tue Apr 8 12:02:24 2014 +0300

       Always use the new dependency generator "engine"
    
        - Replace platform specific find-{requires,provides} scripts with
          trivial wrapper scripts that just call rpmdeps with suitable
          arguments. This way the generated dependencies using the legacy
          external dependency are at least roughly on par with the internal
          depgen as, well, they're generated by the same thing.
        - Changing from find-{requires,provides} scripts to rpmdeps could
          be done by just changing __find_{requires,provides} macros, but
          a lot of packages directly refer to the scripts instead so this
          way we cover far more ground and in an backwards compatible way.

diff --git a/Makefile.am b/Makefile.am
index a6fa619..199cffb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -96,8 +96,7 @@ rpmbin_PROGRAMS = rpm
 bin_PROGRAMS =		rpm2cpio rpmbuild rpmdb rpmkeys rpmsign rpmspec
 
 rpmlibexec_PROGRAMS =
-rpmconfig_SCRIPTS =	autodeps/find-provides autodeps/find-requires \
-			mkinstalldirs \
+rpmconfig_SCRIPTS =	mkinstalldirs \
 			config.guess config.sub
 DISTCLEANFILES = autodeps/find-provides
 DISTCLEANFILES += autodeps/find-requires
diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index 7656c80..f93901e 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -14,6 +14,7 @@ EXTRA_DIST = \
 	perldeps.pl perl.prov perl.req pythondeps.sh osgideps.pl \
 	rpmdb_loadcvt rpm.daily rpm.log rpm.supp rpm2cpio.sh \
 	tcl.req tgpg vpkg-provides.sh \
+	find-requires find-provides \
 	find-requires.php find-provides.php \
 	find-php-provides find-php-requires \
 	mono-find-requires mono-find-provides \
@@ -28,7 +29,7 @@ rpmconfig_SCRIPTS = \
 	brp-strip-shared brp-strip-static-archive \
 	check-files check-prereqs \
 	check-buildroot check-rpaths check-rpaths-worker \
-	find-lang.sh \
+	find-lang.sh find-requires find-provides \
 	perl.prov perl.req perldeps.pl pythondeps.sh osgideps.pl \
 	mono-find-requires mono-find-provides \
 	pkgconfigdeps.sh libtooldeps.sh \
diff --git a/scripts/find-provides b/scripts/find-provides
new file mode 100644
index 0000000..c5cf93b
--- /dev/null
+++ b/scripts/find-provides
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/lib/rpm/rpmdeps --provides
diff --git a/scripts/find-requires b/scripts/find-requires
new file mode 100644
index 0000000..9d192dd
--- /dev/null
+++ b/scripts/find-requires
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/lib/rpm/rpmdeps --requires
